src/main/java/com/hillel/items_exchange/service/UserService.java

```diff
@@ -50,11 +50,20 @@ public boolean existsByEmail(String email) {
         return userRepository.existsByEmail(email);
     }
 
+    public boolean existsByEmailAndPassword(String email, String encodedPassword){
+      return userRepository.findAll().stream()
```

Comment:
Why do you take all users from db instead of talking one by email?

Reply:
Agree. Your way is better. Will be changed.